### ISSUE TITLE
Connectomics in 10 years?

### ISSUE THREAD

**Issue by mrjiaruiwang** [openconnectome/Statistical-Connectomics-Class#184]
What do people think the field will look like in ten years? What are the major milestones/breakthroughs that would need to be met for us to achieve these dreams of the future? Will we be able to better diagnose people based on medical images?

**Comment by adjordan** [comment]
I'm biased, but I think community detection algorithms will continue to improve and be critical in detecting diseased brain regions, such as those found in epilepsy.

**Comment by DSP137** [comment]
I think it's definitely possible that we will be able to diagnose people based on medical images in the future. Not sure how far out that is, but it will be nice if we can diagnose things like Alzheimer's, Parkinson's, Schizophrenia, and other neurologically correlated diseases via brain scans.

**Comment by aceeccc** [comment]
What about for spectral disorders that show a spread of symptoms like autism? Would connectome analysis be able to show anything given that something like autism probably has widely varying neurological causes? (I don't actually know if autistic brains have widely varying connectomes, but I'm just speculating because it is a spectrum disorder)

**Comment by dlee138** [comment]
Although connectomics has great potential, I think we still have a far way to go and diagnosing people based on medical images won't happen within the next 10 years in my opinion.  We still are struggling to define what an "average" brain looks like, let alone a diseased one, and the process of making images in to graphs still has plenty of room for improvement.  I think the next 10 years will bring great progress in the connectomics theory and basic research, but it will take longer than 10 years for apply connectomics to diagnose diseases from brain images on a regular basis.

**Comment by ajulian3** [comment]
I believe connectomics will need to rely on great improvement in their detection algorithms. Specifically, since connectomics is a developing field, it would be beneficial to create a standardization of the data so we could take diagnostic inference. This would be helpful in localizing disease areas of the brain along with further research. How would you suggest they standardize this process?

**Comment by willgray** [comment]
I think that in ten years we will be able to distinguish MRI (mm) scale connectivity properties of normal vs. exceptional (pathological or cognitive, above/below normal)

I think that for higher resolution, we will have a much better understanding of computational motifs/circuits, although not yet at a whole brain level!

**Comment by SandyaS72** [comment]
I agree that 10 years is not probably not enough time to say that we'd be diagnosing disease based on brain images, though I think we'll have made some important strides in integrating information across different imaging and recording modalities in a more computational or automated way, and network-based approaches could play a big role in that.

**Comment by willgray** [comment]
One thing that I try to think about is the imaging modalities themselves.
For example, MRI is non-invasive and seems to be not harmful.  So that's
good.  At a coarse level, it seems plausible to me that we could use this
as a diagnostic tool for humans using the basic infrastructure we have
now.  It's crazy expensive, though.

Electron microscopy means extracting brain tissue, slicing it up into
ultrathin sections and imaging.  So possible I suppose, but much further
from clinical (maybe path?)

Best,
Will

**Comment by itsthefedora** [comment]
This is specific to research in humans, but ... In order to make meaningful progress, I think we're going to need dramatically better measurement apparatuses. Sophisticated black box computational techniques ("deep learning"?) are entering the picture because people are trying to pull out signal from a system whose SNR is extremely low. I strongly doubt that many meaningful conclusions will come out of "connectome" analysis (in humans) that begins in current-generation imaging technology.

If someone created a noninvasive functional neuroimaging device with millisecond temporal / single-unit spatial resolution, the game would be over for vast swaths of the field. That change might not come in 10 years, but it's the change that's going to *really* revolutionize neuroscience research; until then, interpreting (human) connectomics data seems to me like a Rorschach test, more than anything. We're trying to make what we can out of ink blobs.

To conclude: I think in 10 years, connectomics will uncover a hell of a lot about C. Elegans, zebrafish, and maybe rats, on the micro-scale---but not very much about humans. One good experimental setup is worth a hundred exascale clusters "plug-and-chug"-ing deep learning on "-omics" data.
